packages/cli/src/commands/migrate/tasks/config-lint.ts

```diff
@@ -10,7 +13,14 @@ export async function lintConfigTask(): Promise<ListrTask> {
         task.output = `Create .eslintrc.js from config`;
         await _ctx.userConfig.lintSetup();
       } else {
-        task.skip(`Skip creating .eslintrc.js since no custom config is provided`);
+        const configPath = resolve(options.basePath, '.eslintrc.js');
```

Comment:
the gist of these functions is the following:

1. rehearsal provides a minimal configured eslint for typescript as a default
2. [optional] if the repo isn't currently using eslint then we include the dev-dep with eslintrc from step 1
3. [optional] if the repo is already using eslint we extend the eslintrc with the rehearsal eslintrc from step 1
4. [optional] the repo in question is providing a rehearsal userConfig hook for eslint. in this case we want to run the hook AND be sure after installing the userConfig we extend with the eslintrc from step 1

Reply:
Fixed per discussion.